inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp

```diff
@@ -323,7 +323,6 @@ void MKLDNNGraph::InitGraph() {
     SortTopologically();
 
     InitDescriptors();
-    RemoveDroppedEdges();
```

Comment:
for previous int8 impl we removed 2nd and 3rd input since we copy weight into internalBuffer
for dynamism support we must save 3rd input since it used for output shape computation
this stage was added only for this reason so I removed it since we no need it at all